x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_markers.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { AnnotationDomainType, LineAnnotation } from '@elastic/charts';
+import { EuiIcon } from '@elastic/eui';
+import { useWaterfallContext } from '..';
+import { useTheme } from '../../../../../../../observability/public';
+import { euiStyled } from '../../../../../../../../../src/plugins/kibana_react/common';
+
+export function WaterfallCharMarkers() {
+  const { markerItems } = useWaterfallContext();
+
+  const theme = useTheme();
+
+  if (!markerItems) {
+    return null;
+  }
+
+  const markersInfo: Record<string, { label: string; color: string }> = {
+    domContentLoaded: { label: 'DOMContentLoaded', color: theme.eui.euiColorVis0 },
```

Comment:
Should these be translated? They're technical terms, but things like `Largest contentful paint` probably have translatable meanings.